ui/lib/replication/addon/templates/mode/secondaries/add.hbs

```diff
@@ -18,11 +18,14 @@
     </p>
   </div>
   <div class="field">
-    {{!-- TODO fix so it defaults to 30s like in other places or replace with new TTL picker --}}
-    {{ttl-picker onChange=(action (mut ttl)) initialValue="30m" class="is-marginless"}}
-    <p class="help has-text-grey">
-      This is the Time To Live for the generated secondary token. After this period, the generated token will no longer be valid.
-    </p>
+    <TtlPicker2
+      @initialValue="30m"
+      @label="Time to Live (TTL) for generated secondary token"
+      @helperTextDisabled="If not set, the default value (30 minutes) will be used"
+      @helperTextEnabled="After this period, the generated token will no longer be valid."
+      @onChange={{action "updateTtl"}}
+      @changeOnInit={{true}}
```

Comment:
Added this param because I found that if you make a secondary with a custom TTL, and then immediately make another secondary without touching the TTL picker, it uses the previous value instead of the actual default (which is passed here as `initialValue`

Reply:
💨🦁🐑